client/cli/src/commands/runcmd.rs

```diff
@@ -686,7 +684,7 @@ mod tests {
 			"test-id",
 			|| (),
 			vec!["boo".to_string()],
-			Some(TelemetryEndpoints::new(vec![("foo".to_string(), 42)])),
+			TelemetryEndpoints::new(vec![("wss://foo/bar".to_string(), 42)]).ok(),
```

Comment:
is "foo" on its own a valid URL or multiaddr? if it is, we can cancel part of this change :)

Reply:
Not more valid than `wss://foo/bar`

I thought `/wss/foo` will be valid?

Reply:
All right. I took my example from here: https://github.com/paritytech/substrate/blob/5b41575012e935ffefee03d39ae5d925ac7992fd/client/chain-spec/res/chain_spec.json#L18-L20

Guess I'm missing something :p

Reply:
Have no idea also, but according to `url_to_multiaddr` function below probably both variants valid

Reply:
Yes, basically we either provide a valid url or a multiaddr and `url_to_multiaddr` transforms that into a valid multiaddr (when possible :p).
I believe that:
1. `wss://foo/bar` is a valid URL 
2. `/wss/foo` is a valid multiaddr

My question was whether:
```rust
"foo".is_valid_url() || "foo".is_valid_multiaddr()
```